recipe/meta.yaml

```diff
@@ -47,8 +47,6 @@ requirements:
     - httpx
     - redis-server
     - redis-py
-    # https://github.com/openforcefield/openff-bespokefit/issues/162#issuecomment-1108840723
-    - h5py <3.2
```

Comment:
From what I remember there was a [broken version of h5py](https://github.com/conda-forge/h5py-feedstock/issues/103) released which caused issues only when it was imported and we need it to use qcportal. It looks like the 3.2 version has since been removed so the problem may have been fixed so it should be safe to remove the pin.